staging/src/k8s.io/api/admissionregistration/v1alpha1/register.go

```diff
@@ -35,7 +35,7 @@ func Resource(resource string) schema.GroupResource {
 var (
 	// TODO: move SchemeBuilder with zz_generated.deepcopy.go to k8s.io/api.
 	// localSchemeBuilder and AddToScheme will stay in k8s.io/kubernetes.
-	SchemeBuilder      = runtime.NewSchemeBuilder(addKnownTypes)
+	SchemeBuilder      = runtime.NewSchemeBuilder(addKnownTypes, metav1.InstallInGroupVersion(SchemeGroupVersion))
```

Comment:
I don't like these changes until we have a generator for `zz_generated_register.go`.

Reply:
As discussed on slack: we keep `AddToGroupVersion`. So old 3rdparty code is not broken.